### ISSUE TITLE
:wheelchair: a11y(dropdown): options not accessible

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1421]
### Description of this issue

* Options are not read out (screen reader)
* Options cannot be operated with screen reader

### How to fix it

* Possible solution: add select or native behavior 

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hirsch88** [comment]
Select will be deprecated somewhere next year, so we need to check the new component bal-dropdown

**Comment by m4rc0z** [comment]
Compared the Screen Reader result of Design System Dropdown with the one here: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/

To me it seems like this works quiet similar and the options are read when I jump through them.

**Comment by carolabes** [comment]
Hi @hirsch88 & @m4rc0z ,
did you fix bal-select?

FYI: A combobox is used as a combination of input fields and selects like a search. This ticket is about bal-select in forms.
